### ISSUE TITLE
"AND" and "OR" combine query is not working

### ISSUE THREAD

**Issue by sandeepsajan0** [HTTP-APIs/python-hydra-agent#34]
Now, queries can take only "and" or only "or" as operators but if a user tries to give both "and" and "or" operations in a query then the agent is not working.

Actually, we are using set operations `sinter` and `sunion` separately here. So, this issue raised.
Now we have to use them combined for complex queries.
And the Important thing here is to identify that where should be used `sinter` and `sunion`.
I am working on it.

**Comment by xadahiya** [comment]
I think, for now, you should implement something basic like this - 
```
result = rs.Union(
...     rs.Intersection(nirvana, foo_fighters),
...     rs.Intersection(blur, gorillaz)
```
First, we should focus on getting the functionality right then later we can work on optimization and stuff.

Write an algorithm to parse the commands and build this kind of functionality depending on the command.

**Comment by xadahiya** [comment]
You can also use `Redis MULTI` to achieve this. This might help https://robots.thoughtbot.com/redis-set-intersection-using-sets-to-filter-data

**Comment by sandeepsajan0** [comment]
fixed in #44
